recipes/gtk/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set major_minor = "2.24" %}
+{% set version = major_minor + ".31" %}
+
+package:
+  name: gtk
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnome.org/pub/GNOME/sources/gtk+/{{ major_minor }}/gtk+-{{ version }}.tar.xz
+  sha256: 68c1922732c7efc08df4656a5366dcc3afdc8791513400dac276009b40954658
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - perl 5.22.2.1
+    - atk
+    - cairo 1.14.*
+    - fontconfig 2.12.*
+    - freetype 2.7
+    - gdk-pixbuf
+    - gettext
+    - glib 2.51.*
+    - gobject-introspection
+    - graphite2
+    - harfbuzz 1.3.4
+    - icu 58.*
+    - jpeg 9*
+    - libffi
+    - libiconv
+    - libpng >=1.6.28,<1.7
+    - libtiff >=4.0.3,<4.0.8
+    - libxml2 2.9.*
+    - pango 1.40.*
+    - pcre
+    - pixman 0.34.*
+    - xz 5.2.*
+    - zlib 1.2.8
+
+  run:
+    - atk
+    - cairo 1.14.*
+    - fontconfig 2.12.*
+    - freetype 2.7
+    - gdk-pixbuf
+    - gettext
+    - glib 2.51.*
+    - gobject-introspection
+    - graphite2
+    - harfbuzz 1.3.4
+    - icu 58.*
+    - jpeg 9*
+    - libffi
+    - libiconv
+    - libpng >=1.6.28,<1.7
+    - libtiff >=4.0.3,<4.0.8
+    - libxml2 2.9.*
+    - pango 1.40.*
+    - pcre
+    - pixman 0.34.*
+    - xz 5.2.*
+    - zlib 1.2.8
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME             # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME              # [osx]
+    - test -f ${PREFIX}/lib/libgtk-x11-2.0{{ SHLIB_EXT }}     # [linux]
+    - test -f ${PREFIX}/lib/libgtk-quartz-2.0{{ SHLIB_EXT }}  # [osx]
+
+about:
+  home: https://developer.gnome.org/gtk/
+  license: LGPL-2
+  license_file: COPYING
+  summary: Primary library used to construct user interfaces in GNOME applications.
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
Anyone else want to help maintain? I'm afraid I can't promise lots of time to this myself.

Reply:
I can help.